<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrandBrowserDialog</class>
 <widget class="QDialog" name="BrandBrowserDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Review Brands</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>101</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Back</string>
   </property>
   <property name="shortcut">
    <string>Alt+B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pageUpButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>120</y>
     <width>101</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>PGUP</string>
   </property>
   <property name="shortcut">
    <string>PgUp</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pageDownButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>200</y>
     <width>101</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>PGDN</string>
   </property>
   <property name="shortcut">
    <string>PgDown</string>
   </property>
  </widget>
  <widget class="Q3Table" name="brandTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>500</width>
     <height>390</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="numRows">
    <number>8</number>
   </property>
   <property name="numCols">
    <number>4</number>
   </property>
   <row>
    <property name="text">
     <string>*</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>TAG#</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>HD#</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>POS:</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>TRAN# </string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Save</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Delete</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Copy</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>------</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>------</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>------</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>------</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="goTopButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>120</y>
     <width>101</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>TOP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="goBottomButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>200</y>
     <width>101</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>BOTTOM</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">dao/branddao.h</include>
  <include location="global">vector</include>
 </includes>
 <resources/>
 <connections/>
</ui>
